File: c/legal_driving_age.c
#include <stdbool.h>
#include <stdio.h>
#include <stdlib.h>

int main(void) {
  int age = 0;

  printf("What is your age? ");
  scanf("%d", &age);

  if (age < 16) {
    puts("You are not old enough to legally drive.");
  } else {
    puts("You are old enough to legally drive.");
  }

  return EXIT_SUCCESS;
}

File: go/legal_driving_age.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func main() {
	scanner := bufio.NewScanner(os.Stdin)

	fmt.Print("What is your age? ")
	scanner.Scan()
	age, err := strconv.Atoi(scanner.Text())
	if err != nil {
		fmt.Println("enter a valid number")
		os.Exit(0)
	}

	if age < 16 {
		print("You are not old enough to legally drive")
	} else {
		print("You are old enough to legally drive")
	}
}
